docker-compose/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ version: "2"
 
 services:
   alfresco:
-    image: quay.io/alfresco/alfresco-content-repository:7.3.0-M1
+    image: quay.io/alfresco/alfresco-content-repository:7.3.0-A12
```

Comment:
stick to -M versions, -A are alphas

Reply:
also `share` needs to be updated to match repo image